transformation/src/test/scala/org/broadinstitute/monster/hca/HcaPipelineBuilderSpec.scala

```diff
@@ -288,6 +288,8 @@ class HcaPipelineBuilderSpec
 
```

Comment:
@raaidbroad Should the url part of the above `exampleUrlAndFile ` value be a filename instead? It looks like the validateJson method is expecting the name of the metadata file.

Reply:
yep you got it